src/commands.rs

```diff
@@ -320,6 +320,10 @@ pub struct ElectrumOpts {
         default_value = "10"
     )]
     pub stop_gap: usize,
+
+    /// Enable or disable domain validation when connecting to Electrum servers.
+    #[clap(name = "VALIDATE_DOMAIN", long = "validate_domain", takes_value(false))]
+    pub validate_domain: Option<bool>,
```

Comment:
This shouldn't be an `Option`, only a `bool`, if the flag isn't used then the value will default to `false`.
```suggestion
    pub validate_domain: bool,
```

Reply:
oh, Okayy